### ISSUE TITLE
train/test split

### ISSUE THREAD

**Issue by Jbwasse2** [HHTseng/video-classification#20]
From looking at your code it looks like you do a random split over the dataset. However, many of these video clips are from the same video, so wouldn't this pollute your testing dataset?

**Comment by HHTseng** [comment]
yeah, I noticed the problem until you mentioned this now, thank you for that. I may find some time to adjust the splitting and rerun the experiments, but in terms of coding it should work fine otherwise, which was my original intention to build an easy code to show the architecture works.
